semana10/LabeX/labex/src/screen/TripDetails/TripDetails.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect, useState } from 'react'
+import { useHistory, useParams } from 'react-router-dom'
+import {getTripDetail, decideCandidate} from '../../services/requestApi'
+import { goToLogin} from '../../Router/goToPages'
+import {Details,DetailsContent,CandidateInfo,ContainerItem,CandidatesList,BtnTrue,BtnFalse} from './styled'
```

Comment:
aqui ficou bem ruimn a leitura, quando for muitos componentes é melhor quebrar eles pra facilitar, fora que tem algumas coisas que não entendi. como BtnTrue e BtnFalse  hihi

Reply:
realmente ficou meio complicado... BtnTrue e BtnFalse seria os botoes de aprovar ou não um candidato